staticcheck/testdata/src/CheckStructTags/CheckStructTags.go

```diff
@@ -14,13 +14,13 @@ type T1 struct {
 	F int        `json:",omitempty,omitempty"` //@ diag(`duplicate JSON option "omitempty"`)
 	G int        `json:",omitempty,string"`
 	H int        `json:",string,omitempty,string"` //@ diag(`duplicate JSON option "string"`)
-	I int        `json:",unknown"`                 //@ diag(`unknown JSON option "unknown"`)
```

Comment:
Instead of removing the expected diags for `unknown`, use a different, actually unknown option. Here and everywhere else.

Reply:
Done.